get batchIDs from caliber api

jedawson opened the pull request.

jedawson committed:
get batchIDs from caliber api

jedawson committed:
why was it a binary file. hopefully it will update now

jedawson committed:
will this work now

jedawson committed:
add in renamed file

jedawson committed:
delete old file

ratthompson reviewed (approved):
This is a-ok with me

jedawson committed:
rename file

jedawson committed:
fix code smells

jedawson commented:
ok it's ready to merge. i have deleted console logs and fixed the code smells. i tried marking the security hotspot as safe, but i don't have permission to do that.
